src/issue/screens/new-issue.screen.js

```diff
@@ -90,17 +92,19 @@ class NewIssue extends Component {
       ).then(issue => {
         navigation.navigate('Issue', {
           issue,
+          headerLeft: false,
```

Comment:
AH snap, didn't think of that 🤔 

So double tapping the a single tab navigator will take you back to the `root` of that stack because of [this code here](https://github.com/gitpoint/git-point/blob/master/routes.js#L309-L336). However, using `navigation.reset` in this step will cause that to be the root of the stack which we don't really want.

This is a little tricker than I thought it would be 😂. There's an attribute that we should easily be able to extend to the issue screen to prevent swipe backs (https://github.com/react-community/react-navigation/issues/1063#issuecomment-312842038) but I don't want to always disable it for the screen. 

Some thoughts:
1. Maybe we can use this attribute but connect to navigation params so that it's only false based on the param (which we can set only on this navigate step to `Issue Screen`? That way with this + `headerLeft: null` would not allow the user to go back. Sounds easier said than done 🤔 
2. Maybe we can leverage a `modal` screen here instead? Haven't tried to load a modal with React Navigation yet but maybe it might be worthwhile to try here?

Either way, happy to merge this in if you would rather we deal with this as part of a separate bug ticket 🎉